frontend/src/features/videos/VideoCard.tsx

```diff
@@ -192,7 +192,9 @@ function VideoCard({
   const { getCriteriaLabel } = useCurrentPoll();
 
   const videoId = videoIdFromEntity(video);
-  let total_score = 0;
+  const tournesolScore =
+    'tournesol_score' in video ? video.tournesol_score : null;
```

Comment:
As a consequence, the score would not be visible after deployment, until a new ml_train has run.